src/playbackstrategy/msestrategy.js

```diff
@@ -243,15 +251,19 @@ function MSEStrategy (mediaSources, windowType, mediaKind, playbackElement, isUH
   function onBaseUrlSelected (event) {
     const failoverInfo = {
       errorMessage: 'download',
-      isBufferingTimeoutError: false
+      isBufferingTimeoutError: false,
+      code: lastError && lastError.code,
+      message: lastError && lastError.message
```

Comment:
We now have `errorMessage` and `message` in the same object. Is there a way we can distinguish between these two better?

Reply:
Good spot. Digging further, errorMessage is unused, so have removed that too here.